storage/innobase/include/small_vector.h

```diff
@@ -19,8 +19,10 @@ this program; if not, write to the Free Software Foundation, Inc.,
 #pragma once
 /* A normally small vector, inspired by llvm::SmallVector */
 #include "my_global.h"
+#include "my_valgrind.h"
 #include <iterator>
 #include <memory>
+#include <algorithm>
```

Comment:
I see that the header uses macros that are defined in `my_valgrind.h`. But `#include <algorithm>` looks like an overkill. https://en.cppreference.com/w/cpp/utility/move.html says that `std::move` is defined in `<utility>`. Is this about that?